examples/v3/e2e/test/provider.spec.js

```diff
@@ -20,20 +20,18 @@ describe('Pact Verification', () => {
     let token = 'INVALID TOKEN';
 
     return new VerifierV3({
+      logLevel: 'info',
       provider: 'Animal Profile Service V3',
       providerBaseUrl: 'http://localhost:8081',
-
-      requestFilter: (req) => {
+      requestFilter: (req, res, next) => {
```

Comment:
This reminds me that I'm not sure if we are very explicit in the docs about how requestFilter works. I'll open an issue to improve this.

Reply:
Good idea, thanks!